CHANGELOG.md

```diff
@@ -26,7 +26,7 @@
 
 ## Fixed Issues
 
--
+- Fix an error that `if-match:false` is used as default in the header, which is now removed.
```

Comment:
Without context I didn't really understand this comment. Not sure if this is better though:
```suggestion
- Fix setting ETags, when no ETag was specified by sending no `if-match` header instead of `if-match:false`.
```